zeppelin-plugins/launcher/k8s-standard/src/main/java/org/apache/zeppelin/interpreter/launcher/K8sRemoteInterpreterProcess.java

```diff
@@ -275,7 +275,7 @@ Properties getTemplateBindings(String userName) {
     Properties k8sProperties = new Properties();
 
     // k8s template properties
-    k8sProperties.put("zeppelin.k8s.namespace", getNamespace());
+    k8sProperties.put("zeppelin.k8s.interpreter.namespace", getNamespace());
```

Comment:
This line is a breaking change. Please correct your PR description. People need to correct their own `interpreter-spec.yaml`.

Reply:
I think this should be the place where you need to call another method, something like `getInterpreterNamespace()`, where the default value is the output of `getServerNamespace()`.